about.html

```diff
@@ -0,0 +1,116 @@
+            <img id="menu-logo" src="./images/menu.png" alt="">
+            <div class="top-header">
+                <ul>
+                    <li class="single-img"><img src="./images/facebook.png" alt=""></li>
+                    <li class="single-img"><img src="./images/twitter.png" alt=""></li>
+                    <li class="single-top">English</li>
+                    <li class="single-top">My page</li>
+                    <li class="single-top">Logout</li>
+                </ul>
+            </div>
+            <div class="bottom-header">
+                <div class="main-logo">
+                    <a href="index.html" class="logo-link">Code<span>G</span>ate</a>
+                </div>
+                <nav>
+                    <ul>
+                        <li class="single-nav active"><a href="about.html">About</a></li>
+                        <li class="single-nav">Program</li>
+                        <li class="single-nav">Join</li>
+                        <li class="single-nav">Sponsor</li>
+                        <li class="single-nav">News</li>
+                        <li class="single-nav square">CC Campaign</li>
+                    </ul>
+                </nav>
+            </div>
+
+        </header>
+        <div class="mobile-body">
+            <img id="menu-close" src="./images/close.png" alt="">
+            <ul class="nav-mobile">
+                
+                <li><a href="index.html" class="nav-a">Home</a></li>
+                <hr>
+                <li><a href="about.html" class="nav-a">About</a></li>
+                <hr>
+                <li><a href="#!" class="nav-a">Join</a></li>
+                <hr>
+                <li><a href="#!" class="nav-a">Sponsor</a></li>
+                <hr>
+                <li><a href="#!" class="nav-a">News</a></li>
+
+            </ul>
+        </div>
+        <div class="main-content">
+            <h1 class="main-into">We live and breathe software development</h1>
+            <p class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa odio illo tempora corporis quaerat sequi, nam repudiandae? Libero labore praesentium fugiat doloremque distinctio neque dolores odio nam deserunt maiores. Natus explicabo cupiditate deleniti recusandae eius tempore odio sunt, ipsum accusamus.</p>
+            <h3 class="contact">Please contact us per Email for any further questions about CodeGate!</h3>
+            <P class="copy">@codegate@gmail.com</P>
+            
+        </div>
+    </section>
+    <section class="co-about">
+        <div class="co-title">
+            <h3>We live and breathe software development </h3>
+            <hr>
+        </div>
+        <div class="co-description">
+            <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ea reiciendis ut adipisci.</p>
+        </div>
+        
+        <div class="co-container">
+                <img src="./images/logo.png" alt="">
+        </div>
+    </section>
```

Comment:
- [ ] Please make sure that all your images work! Your "Logo" section has problems displaying the image and you didn't include an alternative text to handle these cases. Please consider fixing it! 🤓 🔧 

![image](https://user-images.githubusercontent.com/77069011/219496898-e6b2fd75-3fdc-4126-ae29-ba6677b76f25.png)